addons/viewport/src/Tool.tsx

```diff
@@ -33,6 +28,11 @@ const responsiveViewport: ViewportItem = {
 
 const baseViewports: ViewportItem[] = [responsiveViewport];
 
+const toList = memoize(50)((items: ViewportMap): ViewportItem[] => [
```

Comment:
also surprisingly this was actually wrong

Reply:
@ndelangen so does this 😄

Reply:
What do you mean?

Reply:
https://github.com/storybookjs/storybook/pull/8213#discussion_r328769012